README.md

```diff
@@ -271,6 +271,8 @@ The summary includes the following fields:
 * `sampleOrderStatus2ED10Time`
 * `sampleOrderStatus1UR10`
 * `sampleOrderStatus1UR10Time`
+* `sampleOrderStatus1UR90`
```

Comment:
Please add the other samples in here, too.

Reply:
done